src/opengb/dram/util.cljc

```diff
@@ -0,0 +1,12 @@
+(ns opengb.dram.util)
```

Comment:
Let's keep this in the main namespace. Don't quite want to split things out yet, since this impacts the specs and the specs will probably wander off on their own, along with hiding the fact of the vector versus the record implementation.